controllers/component_build_controller_pac.go

```diff
@@ -871,6 +871,11 @@ func (r *ComponentBuildReconciler) ConfigureRepositoryForPaC(ctx context.Context
 		IsAppInstallationExpected: true,
 	})
 	if err != nil {
+		if strings.Contains(err.Error(), "installation has been suspended") {
+			log.Info(fmt.Sprintf("Installation has been suspended for git repository: %s", repoUrl))
```

Comment:
Logs reader is not in context of the code, so please describe installation of what and where has been suspended. And probably consequences for the Component.